src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataResourceSerializer.cs

```diff
@@ -202,12 +203,13 @@ private void WriteDeltaResource(object graph, ODataWriter writer, ODataSerialize
                 {
                     writer.WriteStart(resource);
                     WriteDeltaComplexProperties(selectExpandNode, resourceContext, writer);
-                    //TODO: Need to add support to write Navigation Links, etc. using Delta Writer
+                    WriteDeltaNavigationProperties(selectExpandNode, resourceContext, writer);
+                    //TODO: Need to add support to write Navigation Links, etc. using Delta Writer.
                     //https://github.com/OData/odata.net/issues/155
                     //CLEANUP: merge delta logic with regular logic; requires common base between ODataWriter and ODataDeltaWriter
                     //WriteDynamicComplexProperties(resourceContext, writer);
                     //WriteNavigationLinks(selectExpandNode.SelectedNavigationProperties, resourceContext, writer);
-                    //WriteExpandedNavigationProperties(selectExpandNode.ExpandedNavigationProperties, resourceContext, writer);
+                    //WriteExpandedNavigationProperties(selectExpandNode, resourceContext, writer);
```

Comment:
Why keep the commented code?

Reply:
I think it is some form of a summary but we can do away with it.

Reply:
What was the decision, to remove or to keep?

Reply:
removed and created issues for the todos